pasystem/pasystem-tool/tool/src/webapp/tools/sakai.pasystem.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<registration>
+    <tool id="sakai.pasystem"
+          title="Public Announcements"
+          description="For managing banner alerts and popup screens">
+
+        <!--  This tool will be made available to the following site types via Site Info > Edit Tools -->
+        <category name="course"/>
```

Comment:
Is the normal configuration that the PA system will be in course sites? I was expecting it to be in the administration workspace by default?